app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
```

Comment:
The `print_name` method is declared to return a `str`, but it does not return any value. It only prints to the console. Consider removing the return type annotation.